lib/spaceship/tunes/tunes_client.rb

```diff
@@ -1,5 +1,10 @@
 module Spaceship
   class TunesClient < Spaceship::Client
+    class << self
+      attr_accessor :login_cache_path
+    end
+    TunesClient.login_cache_path = "/tmp/spaceship_itc_login_url.txt"
```

Comment:
I can do the cleanups.

But first, are you confortable with the idea ? I haven't yet looked at the whole fastlane integration.  it works using eval,right ? and I don't know how eval affects class initialization.

I would like to be sure we don't end up deleting that file too many times in the context of the full integration. I have to play a bit with irb first I guess.